app/views/wikis/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Wiki's</h1>
+
+<div>
+  <ul>
+    <li><%= @wikis %></li>
```

Comment:
We'll go over this when we meet, but making a note so I don't forget...  The trick is to always understand what type of object you're working with.  In this case, the object, `@wikis`, is the collection of wikis (as defined in your controller action).  We need to now loop through that collection and display some data about each one.